### DISCUSSION TITLE
A few questions and upgrade error.

### DISCUSSION THREAD

**Discussion by testmodem** [YetiForceCompany/YetiForceCRM#16475] [General]
Dear Concern .. 

I have a few question and also an issue .. 

1. Can We run multiple versions/instances of yetiforce in vmware, and have them registered with the same name/person detail ?

2. Is there any major difference if I update yetiforce 6.1 to 6.4, than a fresh install of yetiforce 6.4 ? will there both look and function identical ?

3. my issue : After failing to export 'Comments' from 6.1 to 6.4, since we cannot export comments other than programmingg. I decided to upgrade my months old installation of 6.1 to 6.4 but I am stuck when i get to 6.3. I get  this error when it get to 6.3 to 6.4 .. Screen shot attached.

Please advise.

![upgrade history 6 1 - 6 4](https://user-images.githubusercontent.com/59494950/190176541-196c7dcc-b9ce-493f-9be4-c0d574ab8e6b.png)

![6 1 - 6 4 error](https://user-images.githubusercontent.com/59494950/190176482-2dc99a7c-55a8-42f7-a01e-5209e8d440f8.png)

**Comment by bpabiszczak** [comment]
1. Yes. According to the license.
2. There may be differences. For example: in the update, we are not removing fields that are no longer in the new version.
3. Errors are reported to the issue tab [according to the template].

I suppose you are trying hard to try not to update the system properly.

**Reply by testmodem** [reply]
@bpabiszczak  Thank you for your response and clarifying my doubts .
"I suppose you are trying hard to try not to update the system properly." , It maybe due to lack of proper information and knowledge. I will be obliged if you can point me in the right direction. I am new to this and trying to get it going ( 👍
